meetings/notes/sig-meeting-20221115.md

```diff
@@ -0,0 +1,57 @@
+## Meeting Details
+
+- **Date/Time:** [November 15th, 2022 @ 17:00pm UTC / 11:00am ET / 9:00am PT](https://lists.o3de.org/g/o3de-calendar/viewevent?repeatid=39350&eventid=1557573&calstart=2022-11-15)
+- **Location:** [Discord SIG-Network Voice Room](https://discord.gg/62nq7HP5mP)
+- **Moderator:** Pip Potter (lmbr-pip)
+- **Note Taker** Pip Potter (lmbr-pip)
+
+[Agenda Issue](https://github.com/o3de/sig-network/issues/79)
+
+## SIG Updates
+
+### General SIG announcements
+AWS is looking to contribute a [major update](https://github.com/o3de/o3de/issues/13198) to the 3rd person Multiplayer Sample
+
+[O3DCon 2022](https://events.linuxfoundation.org/o3dcon/) talks now on [YouTube](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ). Of interest to SIG:
+* [Building Multiplayer Content for Large Worlds](https://www.youtube.com/watch?v=mY8SgHPDVHA&list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ&index=7)
+* [Scale Testing Multiplayer Games on AWS](https://www.youtube.com/watch?v=QayMbJxOzJ0&list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ&index=17)
+* [Keynote: Supercharging Gameworld Performance Using the Cloud](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ&index=20)
```

Comment:
this link still resolves to the playlist

Reply:
All of the links to specific talks have `&index=x` which resolves to items in the play test. Theres a link to the playlist at the top on purpose.

Lines 16-18 link into the playlist as expected

Reply:
Hmm pasting that link into the browser just resolves to the top level playlist for me (https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ), in both Chrome and Firefox.

Line 17 actually does go directly to the Keynote, but with the playlist available in the right context menu, which seems like the desired behavior. The L17 and L18 links are formatted slightly differently but both contain the playlist & index at the end - maybe you intended to use the same format as L17, with the `watch` arg? : 
![image](https://user-images.githubusercontent.com/34254888/202506475-b7548a51-9ef5-4be1-be1f-e5fdf1560500.png)